cev_eris.dme

```diff
@@ -2499,6 +2498,7 @@
 #include "code\modules\projectiles\guns\projectile\shotgun\regulator.dm"
 #include "code\modules\projectiles\guns\projectile\shotgun\sawnoff.dm"
 #include "code\modules\projectiles\guns\projectile\shotgun\type21.dm"
+#include "code\modules\projectiles\guns\projectile\shotgun\ks23.dm"
```

Comment:
```
#include "code\modules\projectiles\guns\projectile\shotgun\bojevic.dm"
#include "code\modules\projectiles\guns\projectile\shotgun\bull.dm"
#include "code\modules\projectiles\guns\projectile\shotgun\doublebarrel.dm"
#include "code\modules\projectiles\guns\projectile\shotgun\gladstone.dm"
#include "code\modules\projectiles\guns\projectile\shotgun\handmade.dm"
#include "code\modules\projectiles\guns\projectile\shotgun\pump.dm"
#include "code\modules\projectiles\guns\projectile\shotgun\regulator.dm"
#include "code\modules\projectiles\guns\projectile\shotgun\sawnoff.dm"
#include "code\modules\projectiles\guns\projectile\shotgun\type21.dm"
#include "code\modules\projectiles\guns\projectile\shotgun\ks23.dm"
```
It looks like the alphabetic ordering has been broken 👀 

If you "activate" it with Byond Dream Maker, it should be automatically added at the right place.

Reply:
oh i didnt payed attention of an alphabetic order, i can change it right now, i got told to put it in line 2501 some days earlier

Reply:
corrected